application/src/main/java/org/thingsboard/server/config/ThingsboardSecurityConfiguration.java

```diff
@@ -203,6 +217,20 @@ SecurityFilterChain resources(HttpSecurity http) throws Exception {
     }
 
     @Bean
+    @Order(1)
+    SecurityFilterChain httpTransportFilterChain(HttpSecurity http) throws Exception {
+        http
+                .securityMatchers(matchers -> matchers.requestMatchers(DEVICE_API_ENTRY_POINT))
+                .cors(cors -> {})
+                .csrf(AbstractHttpConfigurer::disable)
+                .authorizeHttpRequests(config -> config
+                        .requestMatchers(DEVICE_API_ENTRY_POINT).permitAll())
+                .addFilterBefore(httpTransportRequestSizeFilter(), UsernamePasswordAuthenticationFilter.class);
+        return http.build();
+    }
```

Comment:
I see this is duplicated in the common transport module. Doesn't Spring allow multiple security configurations?